src/pages/EmployeeListing/components/EmployeeDeletePopup.tsx

```diff
@@ -0,0 +1,41 @@
+import { Popup } from "../../../components";
+import { useAppContext } from "../../../store/app.context";
+
+export interface EmployeeDeletePopupProps {
+  employeeId: number;
+  onClose: () => void;
+}
+
+export default function EmployeeDeletePopup({
+  employeeId,
+  onClose,
+}: EmployeeDeletePopupProps) {
+  const appContext = useAppContext();
+
+  function onDelete() {
+    console.log("delete employee", employeeId);
```

Comment:
*all* console.logs must be removed from the application

Reply:
done